docs/release-notes.md

```diff
@@ -3,6 +3,34 @@
 - Table of Contents
 {:toc}
 
+## Version 1.2.0
+
+The 1.2.0 release of MCUboot brings a lot of fixes/updates, where much of the
+changes were on the boot serial functionality and imgtool utility. There are
```

Comment:
If you mention boot serial in here I think we should have an entry for it in "About this release"

Reply:
Now there is!